pages/tracks.tsx

```diff
@@ -0,0 +1,29 @@
+
+
+const Tracks = () => {
+    return (
+      <div className="px-28 py-28">
+          <h1 className="text-5xl">Domains</h1>
+          <ul className="pt-8">
+              <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Web development</h2>
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>UI/UX</h2>
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Blockchain / Web3</h2>
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Artificial Intelligence/Machine Learning</h2>
+                </li>
+                <li className="text-2xl tracking-wide border-2 border-[#005550] py-6 my-6 text-white bg-[#005550] rounded-full text-center">
+                  <h2>Android Development</h2>
+                </li>
```

Comment:
Instead of redundant entries...
Please use the `.map` functionality to maintain D.R.Y format